packages/react-core/src/components/Toolbar/examples/ToolbarColorVariant.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Toolbar, ToolbarItem, ToolbarContent, Button, SearchInput, Checkbox } from '@patternfly/react-core';
+
+export const ToolbarColorVariant: React.FunctionComponent = () => {
+  const [isSecondary, setisSecondary] = React.useState(true);
+
+  return (
+    <React.Fragment>
+      <Toolbar id="toolbar-" colorVariant={isSecondary ? 'secondary' : 'primary'}>
+        <ToolbarContent>
+          <ToolbarItem>
+            <SearchInput aria-label="Items example search input" />
+          </ToolbarItem>
+          <ToolbarItem>
+            <Button variant="secondary">Action</Button>
+          </ToolbarItem>
+          <ToolbarItem variant="separator" />
+          <ToolbarItem>
+            <Button variant="primary">Action</Button>
+          </ToolbarItem>
+        </ToolbarContent>
+      </Toolbar>
+      <Checkbox
+        label="Secondary"
+        isChecked={isSecondary}
+        onChange={(_event, checked) => setisSecondary(checked)}
+        id="isSecondaryCheckbox"
+      />
```

Comment:
Should we add the ability to see all of the variants? Looks like the way it is now toggles between secondary and primary, which we should at least call out that it's secondary when checked, and primary when unchecked, which is different from the default (no value for `colorVariant`). Seems like it might be more clear if we showed primary, secondary, and no-background.

Reply:
I thought of that but followed the drawer model, which has a no-background modifier but only showed an example for secondary - and also I didn't know how to make a dropdown work lol. Maybe I can pair program with someone to do that?

Reply:
@sarah, we should probably update the Drawer example as well.  I agree that we should show all variants.